sirepo/pkcli/service.py

```diff
@@ -263,7 +294,7 @@ def _cfg_ip(value):
 
 
 def _cfg_port(value):
-    return _cfg_int(5001, 32767)(value)
+    return _cfg_int(3000, 32767)(value)
```

Comment:
We'll want to make sure that this 3000 change is documented in the commit stream: allow default React port.

Reply:
What is the best way to add a commit for this line? Should I revert the changes and reapply them with that commit message?